examples/session_controller.rb

```diff
@@ -1 +1,16 @@
-# TODO
+class SessionController < ApplicationController
+  skip_user_authentication only: [:new]
+
+  def new
+  end
+
+  def create
+    if user = Autho::Authentication.new(User, params[:email], params[:password]).user
+      Autho::UserSession.new(self, :user_id, User).user = user
+      redirect_to root_path, flash: { notice: t(:"sessions.successfully_signed_in") }
```

Comment:
Just `redirect_to root_path, notice: …` should be enough. Similarly for `error`.